indico/modules/events/tracks/controllers.py

```diff
@@ -33,6 +42,16 @@ def _process(self):
 
 
 class RHCreateTrack(RHManageTracksBase):
+    def _process(self):
+        form = TrackForm()
+        if form.validate_on_submit():
+            form_data = dict({'event_new': self.event_new}, **form.data)
```

Comment:
We usually pass the parent object in the first argument instead of using `data` for it.